framework/src/main/java/com/jfcbuilder/builders/ChartBuilder.java

```diff
@@ -132,6 +133,30 @@ public ChartBuilder timeData(long[] timeData) {
     return this;
   }
 
+  /**
+   * Overrides the time axis tick label format.
+   * 
+   * @param format The {@link DateFormat} to use
+   * @return Same instance of this builder for chaining method calls
+   * @throws NullPointerException If format is null
+   */
+  public ChartBuilder dateFormat(DateFormat format) {
+    Objects.requireNonNull(format, "Date format is required.");
```

Comment:
With the null checks at line 261 and line 298 null is accepted, so the requirement check here should be removed.